cmd/kritis/kubectl/plugins/resolve/README.md

```diff
@@ -3,46 +3,57 @@
 resolve-tags replaces tagged images in your Kubernetes yamls with their corresponding digests, and prints the new manifest to STDOUT.
 It can be run as a binary or installed as a kubectl plugin.
 
-## kubectl plugin
+## Installation
 
-To install as a kubectl plugin, run:
+## macOS
 
+```shell
+curl -LO https://storage.googleapis.com/resolve-tags/latest/resolve-tags-darwin-amd64.tar && \
```

Comment:
optional nit: for readability, I recommend adusting long pipelines such as this so that the action (| or &&) appear at the beginning of the line, followed by a 2 space indentation, rather than the end of it - and that each part of the pipeline gets it's own line, like:

https://google.github.io/styleguide/shell.xml?showone=Pipelines#Pipelines

Reply:
gotcha I added the indentation but couldn't get the copy/paste in shell to work with the action at the beginning :(